helm/values.yaml

```diff
@@ -24,7 +24,7 @@ service:
 
 nodeLabels: {}
 
-javaOpts: "-XX:InitialRAMPercentage=50.0 -XX:MaxRAMPercentage=75.0"
+javaOpts: "-XX:InitialRAMPercentage=50.0 -XX:MaxRAMPercentage=75.0 -XX:MaxDirectMemorySize=128M"
```

Comment:
Wondering if there's an easy way to figure out the services which don't have this config set ? Might help avoid some future OOMs

Reply:
we mostly override `javaOpts` for all services in a private repo, we could just add this over there.